forem/forem
# Add admin config to always show manually selected users on "Suggested people to follow" list

## What type of PR is this? (check all applicable)


## Description

- Added a config to allow admins to choose to always show "Suggested users" instead of system-generated suggestions in "Suggested people to follow" screen.

I also made some related fixes:
- Fixed some missing whitespace in the description of "Suggested users" input field 
- Refactored some tests related to "Suggested users" and this change to resolve the linting errors I encountered

## Related Tickets & Documents

Closes #9367

## QA Instructions, Screenshots, Recordings

http://localhost:3000/admin/config > Under "All Site Configuration" > Onboarding
- I added a checkbox
![issue_9367](https://user-images.githubusercontent.com/25644062/96378275-c79b3800-11c5-11eb-9317-c892b698e42a.png)

- I don't know if there is an easy way to see the onboarding process again but if you visit http://localhost:3000/users?state=follow_suggestions you can see the list of users in JSON format. You can validate that the list changes according to this config.

## Added tests?


## Added to documentation?
